### DISCUSSION TITLE
Dilemma which to choose Momentum or Daily.dev

### DISCUSSION THREAD

**Discussion by Isaacjon** [dailydotdev/daily#1206] [Feature Requests]
i have been using momentum extension of chrome and it does great job. every time i open new blank tab it shows nice background(pictures of nature or some cool photos) which really impacts my mood but also some really insightful or motivational quotes on the bottom. after i install daily.dev i can't see them anymore(obviously!). is it possible to mix this thing with daily.dev somehow. the thing i am asking is some nice background (photos of nature) and some motivational or insightful quote.

<img width="1680" alt="image" src="https://github.com/dailydotdev/daily/assets/89290880/aae6e4c4-de6b-4dfe-8229-7750507b167c">

**Comment by sabdailydev** [comment]
Thanks for your feedback. 
While we are exploring how to make the feed layout more flexible to users, we don't have plans to implement variations similar to what you have mentioned as of now. We will evaluate it in the future, thanks.
